### DISCUSSION TITLE
Question: Is there maximum limit of API request call for day? Or it can be used unlimited number of rows, lets say 5 Millions query

### DISCUSSION THREAD

**Discussion by RILUCK** [komoot/photon#607] [Q&A] [answered]
For example, Can I request API similar to "photon.komoot.io/api/?q=berlin" for 5+ millions time in a row without exceeding the maximum limit of Photon per day(24 hours). 

Also is there any faster way to process geocoding using Photon, Because just for 2500 queries time taken is more than 40 minutes to process.

**Comment by karussell** [comment, marked as answer]
Why not set up photon on your side? Then you can hit the server without any limits.

> Also is there any faster way to process geocoding using Photon, Because just for 2500 queries time taken is more than 40 minutes to process.

This sounds like a slow response time, which should be also fixed if you set up photon on your own (or use a commercial setup).
